### ISSUE TITLE
Errata records in pulp are created without capitalized severity

### ISSUE THREAD

**Issue by javihernandez** [AlmaLinux/build-system#405]
Report from the user:

```
Dear AlmaLinux support,
 
Recently, if we run "dnf -q updateinfo --updates list sec"
 
ALSA-2024:9317 Unknown/Sec. NetworkManager-1:1.48.10-2.el9_5.alma.1.x86_64
ALSA-2024:9317 Unknown/Sec. NetworkManager-libnm-1:1.48.10-2.el9_5.alma.1.x86_64
ALSA-2024:9317 Unknown/Sec. NetworkManager-team-1:1.48.10-2.el9_5.alma.1.x86_64
ALSA-2024:9317 Unknown/Sec. NetworkManager-tui-1:1.48.10-2.el9_5.alma.1.x86_64
ALSA-2024:9449 Unknown/Sec. bubblewrap-0.4.1-8.el9_5.x86_64
ALSA-2024:9541 Unknown/Sec. expat-2.5.0-3.el9_5.1.x86_64
ALSA-2024:9541 Unknown/Sec. expat-devel-2.5.0-3.el9_5.1.x86_64
…
 
Basically, since around 28 Oct 2024, all update packages is showing "Unknown/Sec" , instead of "Important/Sec." , "Moderate/Sec." , "Low/Sec." in the past , is it normal/expected?
 
Thank you very much for your kind attention.
```

While it isn't a critical problem, we need to fix this problem ASAP.
We need to:
- [x] https://github.com/AlmaLinux/albs-web-server/pull/1085
- [x] Fix the released erratas in pulp
